frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -90,7 +114,9 @@ const isIdirDomainSelected = () => {
 
 function userDomainChange() {
     resetVerifiedUserIdentity();
+    userIdChange();
     formData.value.userId = '';
+    userInfoStep.value.icon = IconSteps.incomplete;
```

Comment:
and this is why I recommend that we shouldn't use local step variables. Because you're sharing the stepState variable with PageTitle component, but you're changing the local step variable, not the stepState variable. Because the stepState is pointing to the reference of your local step variables, it still works, but it's really confused and hard to maintain for long term.

Reply:
can we have a chat about it?

Reply:
yeah, for sure, let's chat about it. Cause I feel probably even there is no need to define it as a global state variable, if we'll pass it as a parameter to the PageTitle component, we can just define that stepState locally in grant access component.